packages/website/src/components/Section.tsx

```diff
@@ -35,16 +46,13 @@ export function Section({ title, children, className, defaultClosed }: SectionPr
 								open: {
 									opacity: 1,
 									height: 'auto',
-									paddingLeft: '1.75rem',
-									paddingRight: '1.75rem',
-									paddingBottom: '2.5rem',
 								},
-								collapsed: { opacity: 0, height: 0, paddingLeft: '1.75rem', paddingRight: '1.75rem', paddingBottom: 0 },
+								collapsed: { opacity: 0, height: 0 },
```

Comment:
This was deliberate, because this was it animates nicely even if they are static compared to the static class on the element.